yarn.lock

```diff
@@ -1289,10 +1289,10 @@
     react-transition-group "1"
     recast "^0.14.4"
 
-"@department-of-veterans-affairs/formation@6.10.3":
-  version "6.10.3"
-  resolved "https://registry.yarnpkg.com/@department-of-veterans-affairs/formation/-/formation-6.10.3.tgz#40fcb6ca083b71e346bf549bf654bb3448ac748b"
-  integrity sha512-7EoM31RJ+mZseo9Eac15zwI1H2ippFCinYAR7nGWtJsg15XHgD+Lo89f/2CzhIsw8+KLgYxmd5Fslp7VXLVr8A==
+"@department-of-veterans-affairs/formation@6.11.0":
+  version "6.11.0"
+  resolved "https://registry.yarnpkg.com/@department-of-veterans-affairs/formation/-/formation-6.11.0.tgz#7a44bea61843d19929d12fc005099e6da44f8f0e"
+  integrity sha512-n/cSKY5SiU5hfeegI2HbUIAcPmS7neMCZk3QT6hmi6pX8SRdelsOTaTz8bsuqSKoww2qbnWenbk7uXj33w6bqA==
```

Comment:
@bkjohnson Was there a merge into https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/ that bumped the Formation version there? I merged a PR a moment ago that did the bump to 6.11. and was surprised that version is already published, https://www.npmjs.com/package/@department-of-veterans-affairs/formation/v/6.11.0

Reply:
We really need to remove access to npm publishing for humans and just let CI do it exclusively

Reply:
I'm late noticing this, but I think it's all been sorted out. I strongly agree with CI publishing though